examples/traefik/docker-compose-swarm-registry.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.7'
+
+services:
+  registry-ui:
+    image: joxit/docker-registry-ui:latest
+    environment:
+      - SINGLE_REGISTRY=true
+      - REGISTRY_TITLE=Docker Registry UI
+      # For deleting images, you need to activate the delete feature in the UI with DELETE_IMAGES=true and in your registry.
+      - DELETE_IMAGES=true
+      - SHOW_CONTENT_DIGEST=true
+      - NGINX_PROXY_PASS_URL=http://registry:5000
+      - SHOW_CATALOG_NB_TAGS=true
+      - CATALOG_MIN_BRANCHES=1
+      - CATALOG_MAX_BRANCHES=1
+      - TAGLIST_PAGE_SIZE=100
+      - REGISTRY_SECURED=true
+      - CATALOG_ELEMENTS_LIMIT=1000
+    networks:
+      - traefik-net 
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+      labels:
+        - "traefik.enable=true"
+        - "traefik.http.routers.registryui.rule=Host(`myregistry.com`)"
+        - "traefik.http.routers.registryui.entrypoints=web"
+        - "traefik.http.services.registryui.loadbalancer.server.port=80"
+        - "traefik.http.routers.registryui.service=registryui"
+    configs:
+      - source: ui_nginx_config
+        target: /etc/nginx/conf.d/default.conf
+
+  registry:
+    image: registry:2.8.3
+    volumes:
+      - /images:/var/lib/registry
+      - /htpasswd:/auth
```

Comment:
You may want to change this to `./images` and `./htpasswd` since we're only testing.